v2/swagger.yaml

```diff
@@ -0,0 +1,629 @@
+swagger: '2.0'
+info:
+  title: Analytics Core API
+  version: '0.2'
+basePath: /api/v2/
+consumes:
+  - application/json
+produces:
+  - application/json
+paths:
+  '/component-analyses/{eco}/{name}/{ver}':
+    get:
+      tags:
+        - Scan Services
+      operationId: f8a_scanner.api_v1.component-analyses
+      summary: Get security and other details about a software component and recommendations to change to a more secured version
+      parameters:
```

Comment:
How requests from  'free tier' and 'already registered tier' of users are differentiated ?

Reply:
Authorization token(UUID) will be added once we are done with Authentication flow design.